deploy/docs/v1_conf_examples.md

```diff
@@ -2,17 +2,25 @@
 
 Until now, Helm users have not been able to modify their Fluentd configuration outside of the specific parameters that we exposed in the `values.yaml` file. Now, we expose the ability to modify the Fluentd configuration as needed. 
 
-Some use-cases include : 
+Some use-cases include: 
  - custom log pipelines, 
  - adding Fluentd filter plugins (ex: fluentd throttle plugin), or 
  - adding Fluentd output plugins (ex: forward to both Sumo and S3)
- - additinal configurantion for sumologic output plugin
+ - additional configuration for sumologic output plugin
 
+**NOTE:** Helm templating like
+```bash
+{{ .Values.fluentd.foo.bar | quote }}
+```
+will **NOT** work if they are specified within one of the below mentioned plugin conf sections, since they are in the `values.yaml` file and are therefore interpreted as literal strings. 
 
 Below you can see a few examples of how this configuration can be set.
 
 ### Custom Log Pipelines
-Now we have exposed an `extraLogs` paramter inside the `logs.containers` section of the `values.yaml` where you can add the output plugin for the custom log pipeline.
+Now we have exposed an `extraLogs` parameter inside the `logs.containers` section of the `values.yaml` where you can add the output plugin for the custom log pipeline.
+
+**NOTE:** This will only send the logs to Sumo if the logs are being collected correctly at the FluentBit level with an input plugin.
+To add a custom endpoint, please follow the steps mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/additional_prometheus_configuration.md#create-a-new-http-source-in-sumo-logic)
```

Comment:
Maybe we should link to the Help docs instead, this might be confusing since we are talking about Prometheus in this section tey might think its for metrics and not for logs?

Reply:
Or maybe we just need a more detailed explanation not sure what the best approach is.

Reply:
Yes, even I was thinking the same. It can be confusing. Have created #627  to expose these configs for helm users. As for now, I think linking the Help docs makes more sense as you suggested Frank.